package.json

```diff
@@ -53,29 +54,32 @@
     "passport": "0.6.0",
     "pg": "^8.6.0",
     "role-acl": "^4.5.4",
+    "semver": "^7.5.2",
     "setup": "^0.0.3",
     "start": "^5.1.0",
     "uglify-js": "^3.15.4",
     "winston": "^3.3.3",
     "yarn": "^1.22.18"
   },
   "devDependencies": {
+    "braces": "^3.0.3",
```

Comment:
This is only needed in dependencies, as devdependencies are the additional packages for development environment and this will try to install it twice, not that it is a big deal but tidy code by removing it from here will look good.

On the other hand, did you tried updating the parent package that has the dependency on this package to a more recent that should be the ideal solution. The reason for that is, now the package.json suggests that this package is required and used directly by the application and in couple of years time if the parent package is removed and not used, this will still be lying around and no one would know what to do with it.

Reply:
@hemantgoswami , removed these packages from package.json file now. please resolve and approve PR.